references/js-callout/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "js-callout",
+  "scripts": {
+    "lint": "../../node_modules/.bin/eslint -c ../../.eslintrc-jsc.yml .",
+    "deploy": "./node_modules/.bin/apigeetool deployproxy -u $APIGEE_USER -p $APIGEE_PASS -o $APIGEE_ORG -e $APIGEE_ENV -n js-callout-v1 -V",
```

Comment:
we can use `-n $npm_package_name` so that it is read from the `name` of package.json. Saves a duplication

Reply:
your proposal is actually more characters to type for no apparent change in the outcome. So no... :)